src/services/auth.services.ts

```diff
@@ -38,7 +38,13 @@ export default class AuthServices {
     return axios.get(url).then(response => {
       if (response?.data) {
         return {
-          contacts: response.data.contacts,
+          contacts: response.data.contacts
+            // map phoneExtension -> extension in every element
+            .map(contact => ({
+              email: contact.email,
+              phone: contact.phone,
+              extension: contact.phoneExtension
+            })),
```

Comment:
So this service does the translation between the app's schema and the API's schema.